src/Psalm/Config.php

```diff
@@ -1090,6 +1095,9 @@ private static function fromXmlAndPaths(
             $attribute_text = (string) $config_xml['findUnusedCode'];
             $config->find_unused_code = $attribute_text === 'true' || $attribute_text === '1';
             $config->find_unused_variables = $config->find_unused_code;
+        } elseif (!defined('__TESTING__')) {
```

Comment:
Seems a weird name

Reply:
How about `__IS_TEST_ENV__`?

Reply:
Oh, I misread what it was supposed to do. I thought it displayed only in test env (but it's the opposite)

Why did you go to the trouble of excluding that for tests? Was it breaking tests?

Reply:
Tests are set to fail if there was any data sent to output streams.